### ISSUE TITLE
FrontEnd: Create new user screen

### ISSUE THREAD

**Issue by laredotornado** [chicommons/maps#24]
We need a simple login screen, using the styles from our other forms, that will allow someone to create a new user in the system.  The fields should be

First name
last name
Username
Password
Confirm Password
Email

The form should submit via POST to the "/users/create" endpoint and the body of the form should be JSON and look like

{
    "first_name": "Dave",
    "last_name": "Example",
    "username": "exampleuser",
    "password": "examplepass",
    "email": "dave@example.com"
}

**Comment by kevinjung2** [comment]
I currently have the page created on local and just need the backend endpoint to be created to test the frontend functionality. Mainly to make sure error handling is working.

**Comment by kevinjung2** [comment]
waiting on https://github.com/chicommons/maps/issues/25 @laredotornado

**Comment by laredotornado** [comment]
Hi @kevinjung2 , pushed backend supporting code to the authentication branch.  But note the endpoint URL is slightly different -- instead of POSTing to "/users/create", you will want to POST to just "/users/".  An example script with a curl request can be found in web/scripts/test_post_create_user.sh .  Let me know if you bump into any issues.

Thanks! -

**Comment by SteveEdiger** [comment]
Kevin and Dave will get together on Sunday.

**Comment by kevinjung2** [comment]
I pushed the code that adds the new user screen to the authentication branch. It is not fully finished but at this point I think it will be best to create a new ticket to handle the rest which is just error handling.

**Comment by laredotornado** [comment]
Hi Kevin! I created a couple of other tickets to address error handling

https://github.com/chicommons/maps/issues/181
https://github.com/chicommons/maps/issues/182

Let me know if you think they need more detail. - Dave
